src/iconsManifest/supportedExtensions.ts

```diff
@@ -837,7 +837,8 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'bun',
-      extensions: ['bun', 'lockb', 'bun.lock'],
+      extensions: ['bun', 'bun.lockb', 'bun.lock'],
```

Comment:
I don’t think there’s a file named `bun`, is there? I do see there are [`bunfig.toml` and `.bundig.toml`](https://bun.sh/docs/runtime/bunfig). You might as well add those while at it.

```suggestion
      extensions: ['.bunfig.toml', 'bunfig.toml', 'bun.lockb', 'bun.lock'],
```

Also Bun for Visual Studio Code adds the [`bun.lockb`](https://github.com/oven-sh/bun/blob/b5b51004e8de80dcd43849e4da1d2623ee3d3008/packages/bun-vscode/package.json#L288) language ID. You could register the icon for that too, so it shows up in the language picker.

Reply:
I'm not sure about how to config the language ID as you mentioned, please help!

Reply:
Done 👍